### PR TITLE
update developer guide with new ovirt-js-dependencies instructions

### PR DESCRIPTION
**Pull request by gregsheremeta**
Fixes issue https://gerrit.ovirt.org/#/c/61716/

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @gregsheremeta

@sandrobonazzola

### CODE REVIEW

File: source/develop/developer-guide/engine/engine-development-environment.html.md [hunk 1]
```diff
@@ -20,131 +20,140 @@ The purpose of this page is primarily to align the community experience with the
...
+                # yum install http://resources.ovirt.org/pub/yum-repo/ovirt-releaseXY.rpm
 
-<b>NOTE:</b> This does not work for master because it is available after release.
+        <b>NOTE:</b> This does not work for master because it is available after release.
 
-###### Manually
+    * Manually
 
-Create the file `/etc/yum.repos.d/ovirt-snapshots.repo`, and replace `@distro@` in the following code block with `fc` for Fedora or `el` for RHEL or an equivalent distribution.
+        Create the file `/etc/yum.repos.d/ovirt-snapshots.repo`, and replace `@distro@` in the following code block with `fc` for Fedora or `el` for RHEL or an equivalent distribution.
```
Comment (rollandf) [inline source/develop/developer-guide/engine/engine-development-environment.html.md]: The repo block disappeared.

Comment (gregsheremeta) [inline source/develop/developer-guide/engine/engine-development-environment.html.md]: We were going to use 'tested master' repo, and that gets installed automatically via the above command 'yum install http://resources.ovirt.org/pub/yum-repo/ovirt-releaseXY.rpm'

However, people didn't want to switch to tested master, so I'll revert that stuff.

Comment (gregsheremeta) [inline source/develop/developer-guide/engine/engine-development-environment.html.md]: Correction -- the tested master block is added automatically on line 45

Comment (gregsheremeta) [inline source/develop/developer-guide/engine/engine-development-environment.html.md]: Correction 2 - yeah, it shouldn't have disappeared. It was for manual installation. Fixed.

**Comment by gregsheremeta** [comment]
@sandrobonazzola Can you take a look?

**Comment by gregsheremeta** [comment]
Here's a print of the page after these changes.
[oVirt Engine Development Environment.pdf](https://github.com/oVirt/ovirt-site/files/876132/oVirt.Engine.Development.Environment.pdf)

File: source/develop/developer-guide/engine/engine-development-environment.html.md [hunk 2]
```diff
@@ -20,131 +20,127 @@ The purpose of this page is primarily to align the community experience with the
...
-      gpgcheck=0
-      priority=10
+          [ovirt-snapshots-static]
+          name=local
+          baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot-static/rpm/@distro@$releasever
+          enabled=1
+          gpgcheck=0
+          priority=10
 
-##### Set Up the PatternFly Repository
```
Comment (rollandf) [inline source/develop/developer-guide/engine/engine-development-environment.html.md]: PatternFly is not needed anymore ?

Comment (rollandf) [inline source/develop/developer-guide/engine/engine-development-environment.html.md]: Other than that looks good

Comment (gregsheremeta) [inline source/develop/developer-guide/engine/engine-development-environment.html.md]: Correct, patternfly is delivered via ovirt-js-dependencies. It's one of 5 or 6 libraries now, and we're just packaging them all up in there.

**Pull request merged**